### PR TITLE
Lavaland Xeno Nest Fix 1.5

### PR DESCRIPTION
**Pull request by ishitbyabullet**
## About The Pull Request
Anturk asked me to make a subtype of the xeno wall. I forgot to do it in time before it got merged because I had a really fun weekend.

Maintainers probably shouldn't give me GBP for this. The main issue is technically fixed so this is just a bonus.

## Why It's Good For The Game
I don't even know why since I forgot to ask. Consider this PR as like dlc if it is necessary.

## Changelog
:cl:
fix: The lavaland xeno nest is still fixed.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/aliens.dm [hunk 1]
```diff
@@ -92,6 +92,11 @@
 /obj/structure/alien/resin/wall/BlockSuperconductivity()
 	return 1
 
+/// meant for one lavaland ruin or anywhere that has simplemobs who can push aside structures
+/obj/structure/alien/resin/wall/immovable
+	desc = "Dense resin solidified into a wall."
+	move_resist = 3000
```
Comment (Fikou) [inline code/game/objects/structures/aliens.dm:98]: use a define

Comment (ishitbyabullet) [reply code/game/objects/structures/aliens.dm:98]: what?

**Review by EOBGames — APPROVED** [review]
I mean, the map changes are perfectly fine- not sure if I'm cool to merge it with the code change, though, so I'll leave that to an actual maint

**Comment by Mothblocks** [comment]
cowerd

**Pull request merged**